src/index.css

```diff
@@ -1,13 +1,71 @@
+* {
```

Comment:
How did you decide on what to put in index.css versus global styling component?

Reply:
Well spotted! Thank you for making me aware of that. I don't plan to use it and forgot to remove the CSS file. 😅